src/hooks/useDownloadFile.ts

```diff
@@ -0,0 +1,35 @@
+import { addToast } from '~/core/apolloClient'
+
+export const useDownloadFile = () => {
```

Comment:
As a next step
Can we plan a refactor to centralise methods from this file and `src/core/utils/downloadFiles.ts`, and check why `openNewTab` there is exported? IMO it shouldn't 
We can create a Linear improvement or tackle right away

Reply:
I think this task is outside the scope of this feature but definitely we need to tackle it